.vscode/extensions.json

```diff
@@ -2,14 +2,15 @@
   // See https://go.microsoft.com/fwlink/?LinkId=827846
   // for the documentation about the extensions.json format
   "recommendations": [
-    "esbenp.prettier-vscode",
     "biomejs.biome",
-    "ms-python.python",
-    "ms-python.black-formatter",
     "dbaeumer.vscode-eslint",
+    "esbenp.prettier-vscode",
     "lextudio.restructuredtext",
-    "ziyasal.vscode-open-in-github",
+    "mkhl.direnv",
+    "ms-python.black-formatter",
+    "ms-python.python",
     "timonwong.shellcheck",
-    "tyriar.sort-lines"
```

Comment:
sorted, and added `mkhl.direnv`

This is more important for getsentry. In getsentry we use direnv to find the sentry folder and do some stuff. Added here to for consistency.